### PR TITLE
Fixes for MediaEmbeds

### PR DESCRIPTION
**Pull request by pwnsdx**

### CODE REVIEW

File: test/unit_tests/media/MediaEmbedsSpec.js [hunk 1]
```diff
@@ -127,73 +127,80 @@ describe('MediaEmbeds', function() {
...
         const message = build_message_with_anchor(link);
 
         expect(z.media.MediaParser.render_media_embeds(message)).toBe(message);
       });
 
-      it('https://www.youtube.com/playlist?list=PLNy867I3fkD6LqNQdk5rAPb6xAI-SbZOd', function() {
-        const link = 'https://www.youtube.com/playlist?list=PLNy867I3fkD6LqNQdk5rAPb6xAI-SbZOd';
+      it('does not render a malicious youtube link', function() {
+        const link = 'https://xn--yutube-wqf.com/#youtu0be/v/fKopy74weus',
+              message = build_message_with_anchor(link);
```
Comment (bennycode) [inline test/unit_tests/media/MediaEmbedsSpec.js]:
```js
const link = 'https://xn--yutube-wqf.com/#youtu0be/v/fKopy74weus';
const message = build_message_with_anchor(link);
```

File: app/script/media/MediaEmbeds.js [hunk 2]
```diff
@@ -89,10 +89,12 @@ z.media.MediaEmbeds = (function() {
    * @returns {string} Youtube embed URL
   */
   const _generate_youtube_embed_url = function(url) {
-    if (url.match(/youtu.be|youtube.com/)) {
+    if (url.match(_regex.youtube)) {
 
       const video_id = url.match(/(?:embed\/|v=|v\/|be\/)([a-zA-Z0-9_-]{11})/);
-      if (!video_id) return;
+      if (!video_id) {
+          return;
```
Comment (lipis) [inline app/script/media/MediaEmbeds.js]: indentatin

Comment (lipis) [reply app/script/media/MediaEmbeds.js]: you should get a plugin for editorconfig for your editor: http://editorconfig.org/

Comment (ghost) [reply app/script/media/MediaEmbeds.js]: I tried to use https://marketplace.visualstudio.com/items?itemName=lonefy.vscode-JS-CSS-HTML-formatter unfortunately it started to reindent everything so I disabled it, I'll use editorconfig then, thanks.

**Pull request merged**